clients/drcachesim/tools/record_filter_launcher.cpp

```diff
@@ -128,8 +128,19 @@ static droption_t<uint64_t> op_trim_after_timestamp(
 droption_t<bool> op_encodings2regdeps(
     DROPTION_SCOPE_FRONTEND, "filter_encodings2regdeps", false,
     "Enable converting the encoding of instructions to synthetic ISA DR_ISA_REGDEPS.",
-    "This option is for -simulator_type record_filter. When present, it converts "
+    "This option is for -tool record_filter. When present, it converts "
     "the encoding of instructions from a real ISA to the DR_ISA_REGDEPS synthetic ISA.");
+
+/* XXX i#6369: we should partition our options by tool. This one should belong to the
```

Comment:
This comment doesn't really make sense here as this is a single tool: should be removed.

Reply:
Got it, removed, also from the other file.

Reply:
My bad, reintroduced it in options.cpp.